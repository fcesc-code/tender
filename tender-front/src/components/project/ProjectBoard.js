import React from 'react';
import { Link } from 'react-router-dom';


function ProjectBoard() {
  const project = props.project;
  return (
    <li key={project.title}>
      <div className="project__board">
        <h3>{project.title}</h3>
        <h2>Status: {project.status}</h2>
        <Link to={project.slug}>Find out more of this project</Link>    
        <p>Description: {project.description}</p> 
        <p>Location: {project.location.street}, {project.location.city}, {project.location.postalCode}</p>
        <p>{project.location.region}{project.location.country}</p>
      </div>
    </li>
  );
}

export default ProjectBoard;