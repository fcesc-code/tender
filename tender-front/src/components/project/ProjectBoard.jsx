import React from 'react';
import DonutChart from '../graphics/DonutChart';
//import { Link } from 'react-router-dom';

function ProjectBoard() {
  return (
    <>
      <h3>Budgets' summary</h3>
      <DonutChart />
    </>
  );
}

export default ProjectBoard;



/*
  const project = props.project;
  return (
    <li key={project.title}>
      <div className="project__board">
        <h3>{project.title}</h3>
        <h2>Status: {project.status}</h2>
        <Link to={project.slug}>Find out more of this project</Link>    
        <p>Description: {project.description}</p> 
        <p>Location: {project.location.street}, {project.location.city}, {project.location.postalCode}</p>
        <p>{project.location.region}{project.location.country}</p>
      </div>
    </li>
  );
*/