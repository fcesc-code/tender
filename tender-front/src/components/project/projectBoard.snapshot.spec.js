import React from "react";
import renderer from "react-test-renderer";
import ProjectBoard from "./ProjectBoard.jsx";

const tree = renderer.create(<ProjectBoard />).toJSON();

describe("ProjectBoard component snapshot", () => {
  test("should match", () => {
    expect(tree).toMatchSnapshot();
  });
});
