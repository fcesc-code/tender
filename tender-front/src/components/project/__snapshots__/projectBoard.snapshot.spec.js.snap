// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectBoard component snapshot should match 1`] = `
<h3>
  Sample for the project board
</h3>
`;
